tests/test_assembly.py

```diff
@@ -399,7 +398,7 @@ def get_doc_nodes(doc, leaf=False):
 
         name_att = TDataStd_Name()
         label.FindAttribute(TDataStd_Name.GetID_s(), name_att)
-        name = TCollection_ExtendedString(name_att.Get()).ToExtString()
```

Comment:
What is happening here?

Reply:
This was an unused variable.  I'll clean it up and remove the line instead of comment out.